ts/components/debug/DebugPrettyPrint.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+WARNING: This component is not referenced anywhere, but is used
+for development purposes. for development purposes. Don't REMOVE it!
+*/
+import {
+  HStack,
+  IOColors,
+  IconButton,
+  LabelSmall,
+  useTypographyFactory
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import { Prettify } from "../../types/helpers";
+import { clipboardSetStringWithFeedback } from "../../utils/clipboard";
+import { withDebugEnabled } from "./withDebugEnabled";
+
+type ExpandableProps =
+  | {
+      expandable: true;
+      isExpanded?: boolean;
+    }
+  | {
+      expandable?: false;
+      isExpanded?: undefined;
+    };
+
+type Props = Prettify<
+  {
+    title: string;
+    data: any;
+  } & ExpandableProps
+>;
+
+/**
+ * This component allows to print the content of an object in an elegant and readable way.
+ * and to copy its content to the clipboard by pressing on the title.
+ * The component it is rendered only if debug mode is enabled
+ */
+export const DebugPrettyPrint = withDebugEnabled(
+  ({ title, data, expandable = true, isExpanded = false }: Props) => {
+    const [expanded, setExpanded] = React.useState(isExpanded);
+    const prettyData = React.useMemo(
+      () => JSON.stringify(data, null, 2),
+      [data]
+    );
+
+    const content = React.useMemo(() => {
+      if ((expandable && !expanded) || !expandable) {
+        return null;
+      }
+      return (
+        <View style={styles.content} pointerEvents="box-only">
+          <CustomBodyMonospace>{prettyData}</CustomBodyMonospace>
+        </View>
+      );
+    }, [prettyData, expandable, expanded]);
+
+    return (
+      <View testID="DebugPrettyPrintTestID" style={styles.container}>
+        <View style={styles.header}>
+          <LabelSmall weight="Bold" color="white">
+            {title}
+          </LabelSmall>
+          <HStack space={16}>
+            {expandable && (
+              <IconButton
+                icon={"copy"}
+                accessibilityLabel="copy"
+                onPress={() => clipboardSetStringWithFeedback(prettyData)}
+                color="contrast"
+              />
+            )}
+            {expandable && (
+              <IconButton
+                icon={expanded ? "chevronTop" : "chevronBottom"}
+                accessibilityLabel="expand"
+                onPress={() => setExpanded(_ => !_)}
+                color="contrast"
+              />
+            )}
```

Comment:
I removed this check since it was not intended, good catch! Thank you. 
Addressed here: https://github.com/pagopa/io-app/pull/6055/commits/9fc67797da383d89cf18324f98d8532b316c7441